gems/pending/OpenStackExtract/MiqOpenStackVm/MiqOpenStackImage.rb

```diff
@@ -2,6 +2,7 @@
 require_relative '../../MiqVm/MiqVm'
 
 class MiqOpenStackImage
+  attr_reader :vmConfigFile
```

Comment:
Can this conform to snake case naming, or does it need to stay camel case for api compatibility?